docs/content/docs/how-to/creating-tables.md

```diff
@@ -114,6 +114,79 @@ Partition keys must be a subset of primary keys if primary keys are defined.
 By configuring [partition.expiration-time]({{< ref "docs/maintenance/manage-partition" >}}), expired partitions can be automatically deleted.
 {{< /hint >}}
 
+## Create Table As
+
+Table can be created and populated by the results of a query, for example, we have a sql like this: `CREATE TABLE table_b AS SELECT id, name FORM table_a`,
+The resulting table `table_b` will be equivalent to create the table and insert the data with the following statement:
+`CREATE TABLE table_b (id INT, name STRING); INSERT INTO table_b SELECT id, name FROM table_a;`
+
+{{< tabs "create-table-as" >}}
+
+{{< tab "Flink" >}}
+
+// For streaming mode, you need to enable the checkpoint.
```

Comment:
Could this use NOTE to explain? The `//` looks no good to me.

Reply:
I update it to /* xxxxx */, I found it is fine on my computer, what about you?

Reply:
LGTM.